client/src/App.js

```diff
@@ -16,9 +16,9 @@ const Card = styled.div`
 function App() {
   return (
     <Card>
-      <Input placeholder="Name" />
-      <Input placeholder="Email" />
-      <Input placeholder="Password" />
+      <Input placeholder="Name" id="name" type="text" />
+      <Input placeholder="Email" id="email" type="text" />
+      <Input placeholder="Password" id="password" type="password" />
```

Comment:
For a draft input form, this looks good. But I think you do not need the `id=` on the `Input`component because I remember that when you want to create forms in React, you create controlled components which update their state with an `onChange` and the submit button then submits the current state (with the values from all three inputs) directly and you do not need an `id` to access the values.

Hope you understand my strange explanation ;-) For questions, write me on Slack.

https://reactjs.org/docs/forms.html

Reply:
Good point! Tnx 👍